src/ui/components/DownloadsManagerView/DownloadItem.tsx

```
@@ -10,6 +11,18 @@ import FileIcon from './FileIcon';
 
 type DownloadItemProps = Download & ComponentProps<typeof Box>;
 
+const rowStyles = css`
+  border-block-start: 1px solid
+    var(--rcx-color-stroke-extra-light, transparent);
```

ESLint

[error] Delete `⏎···`

(prettier/prettier)